server/plugin/api.go

```diff
@@ -130,6 +134,7 @@ func (p *Plugin) initializeAPI() {
 
 	apiRouter.HandleFunc("/config", checkPluginRequest(p.getConfig)).Methods(http.MethodGet)
 	apiRouter.HandleFunc("/token", checkPluginRequest(p.getToken)).Methods(http.MethodGet)
+	apiRouter.HandleFunc("/settings", p.getSettings).Methods(http.MethodGet)
```

Comment:
Could you please move this after `apiRouter.HandleFunc("/connected", p.getConnected).Methods(http.MethodGet)` to keep the list sorted by middle ware type?

Reply:
done.

Reply:
It seems like that change wasn't pushed. Could you please double check?

Reply:
Sorry @hanzei for late reply. I have fixed the ordering.